frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -143,16 +143,18 @@ function filemanagerutil.genResetSettingsButton(file, caller_callback, button_di
                     BD.filepath(file)),
                 ok_text = _("Reset"),
                 ok_callback = function()
-                    local FileManager = require("apps/filemanager/filemanager")
-                    local ui = FileManager.instance
-                    if not ui then
-                        local ReaderUI = require("apps/reader/readerui")
-                        ui = ReaderUI:_getRunningInstance()
+                    local custom_metadata_updated = filemanagerutil.purgeSettings(file)
+                    if custom_metadata_updated then -- refresh coverbrowser cached book info
```

Comment:
Bad naming of the local variable ?

Reply:
Oh, ok, I see you named it that way in `DocSettings:purge()`.
(Feels a bit odd that :purge() would return "custom_metadata_updated" - I guess it means "custom_metadata_removed" ?)

(Nice to see that you're planning for the future with a generic "custom metadata" :)